src/main/resources/additional_properties.json

```diff
@@ -18,5 +18,365 @@
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        {
+          "type": "object",
+          "title": "Unique Key Debezium Topic Router (enforcing message key uniqueness)",
+          "description": "Debezium Topic Router for sending change events from multiple physical tables to a specific topic and enforcing message key uniqueness. See: <a href=\"https://debezium.io/documentation/reference/transformations/topic-routing.html\">Topic Routing section</a> in the Debezium docs.",
+          "required": ["debezium_topic_router_unique_keys"],
+          "additionalProperties": false,
+          "properties": {
+            "debezium_topic_router_unique_keys": {
+              "additionalProperties": false,
+              "required": ["topic_regex", "output_topic", "key_field_name"],
+              "properties": {
+                "topic_regex": {
+                  "title": "Input topic regex",
+                  "description": "A regular expression on the original/input destination topic name of each message that determines if it should be routed to a another destination topic and captures one or more groups of characters to be applied in <code>output_topic</code>",
+                  "type": "string",
+                  "format": "regex"
+                },
+                "output_topic": {
+                  "title": "Output topic",
+                  "description": "A regular expression replacement string that defines the destination topic name based on the captured groups from <code>topic_regex</code> .",
+                  "type": "string"
+                },
+                "key_field_name": {
+                  "title": "Key field name",
+                  "description": "Name of the field to be added to the message key. The value of this field identifies the original table name using the original/input destination topic name and can optionally be manipulated with <code>key_field_topic_regex</code> and <code>key_field_value</code>.",
+                  "type": "string",
+                  "default": "__dbz__physicalTableIdentifier"
+                },
+                "key_field_topic_regex": {
+                  "title": "Key field topic regex",
+                  "description": "Regular expression that gets applied on the original/input destination topic name to capture one or more groups of characters and then apply them in <code>key_field_value</code>",
+                  "type": "string",
+                  "format": "regex",
+                  "default": "(.*)"
+                },
+                "key_field_value": {
+                  "title": "Key field value",
+                  "description": "A regular expression replacement string that sets the value of the inserted key field based on the captured groups from <code>key_field_topic_regex</code>.",
```

Comment:
```suggestion
                  "description": "A regular expression that specifies a replacement string to insert into the key field name. The expression is based on the names of the groups specified by the <code>key_field_topic_regex</code> field.",
```

Reply:
same as above @roldanbob

Reply:
I think also here the description is not close anymore to what is technically happen.